app/directives/proposal/proposal.directive.html

```diff
@@ -23,14 +23,13 @@
             <pre class="db-margin-05-top db-transparent" ng-if="c.mode === c.modes.OPEN" ng-bind-html="c.uiProposal.text | unsafe"></pre>
             <pre id="editor" class="db-margin-05-top" ng-if="c.mode === c.modes.EDIT" contenteditable="true" ng-model="c.editedText"></pre>
 
-
             <div ng-if="c.mode !== c.modes.COLLAPSED" class="db-flex-right db-flex-area">
-                <button ng-if="c.uiProposal.isOwnProposal && (c.mode !== c.modes.EDIT)" class="btn btn-danger db-btn-big db-margin-2" ng-disabled="false"
-                    ng-click="c.remove()"><span translate translate-cloak>STR_DELETE</span></button>
-                <button ng-if="c.uiProposal.isOwnProposal && !c.uiProposal.isPublished && (c.mode !== c.modes.EDIT)" class="btn btn-success db-btn-big db-margin-2" ng-disabled="false"
-                    ng-click="c.send()"><span translate translate-cloak>STR_PUBLISH</span></button>
-                <button ng-if="c.uiProposal.isOwnProposal && !c.uiProposal.isPublished && (c.mode !== c.modes.EDIT)" class="btn btn-primary db-btn-big db-margin-2" ng-disabled="false"
-                    ng-click="c.edit()"><span translate translate-cloak>STR_EDIT</span></button>
+                <button ng-if="c.uiProposal.isOwnProposal && (c.mode !== c.modes.EDIT)" class="btn btn-danger db-btn-big db-margin-2" db-confirm confirm-config="c.deleteConfig"
+                    ng-disabled="false" ng-click="c.remove()"><span translate translate-cloak>STR_DELETE</span></button>
```

Comment:
Old code but should `ng-disabled="false"` be removed? Ok to do it in a separate pr/commit.